README.md

```diff
@@ -1591,6 +1612,12 @@ $ rm -rf .git/modules/submodulename
 
 ## Miscellaneous Objects
 
+### Copy a folder or file from one branch to another
+
+```sh
+$ (my-branch)$ git checkout <branch-you-want-the-directory-from> -- <folder-name or file-name>
```

Comment:
Remove `$ (my-branch)`. It isn't necessary, and you've mistakenly added two dollar signs. Thanks.

Reply:
sure

Reply:
done